libs/translator/reader/reader.cpp

```diff
@@ -1039,8 +1039,8 @@ void UsdArnoldReader::InitCacheId()
     UsdStageCache::Id id = stageCache.Insert(_stage);
     // now our reader will have a cacheID
     _cacheId = id.ToLongInt();
-    // there's one ref count for this cache ID, for the current proc
-    s_cacheRefCount[_cacheId] = 1; 
+    // stageCache.Insert can return an existing stage, so we increase the ref count for that stage in case it exists
+    s_cacheRefCount[_cacheId]++; 
```

Comment:
are we sure the first time we add an entry in the map by doing s_cacheRefCount[_cacheId] , the value will be initialized to 0 ? I would think this code would trigger an uninitialized variable.

Wouldn't it be safer to write it like :
```
if (s_cacheRefCount.find(_cacheId) == s_cacheRefCount.end())
    s_cacheRefCount[_cacheId] = 1;
else
    s_cacheRefCount[_cacheId]++;


```

Reply:
yes, `s_cacheRefCount[_cacheId]` will default value initialize, which is 0.

Reply:
ok cool